openfisca_us/parameters/tax/payroll/FICA/medicare/additional

```diff
@@ -0,0 +1,14 @@
+description: The Additional Medicare Tax rate is applied for earnings above an exclusion.
+rate:
+  2013-01-01: 0.009
+exclusion:
+  SINGLE:
+    2013-01-01: 200_000
```

Comment:
@nikhilwoodruff wdyt about making this `_` a standard practice?

Reply:
I like this, no downsides IMO, it's just more readable. Plus, it'll be trivial to convert all existing parameters into this format if we go with it.